README.md

```diff
@@ -113,6 +113,11 @@ Default: None
 Specifies the number of free IP addresses that the `ipamD` daemon should attempt to keep available for pod assignment on the node\. For example, if `WARM_IP_TARGET` is set to 10, then `ipamD` attempts to keep 10 free IP addresses available at all times\. If the elastic network interfaces on the node are unable to provide these free addresses, `ipamD` attempts to allocate more interfaces until `WARM_IP_TARGET` free IP addresses are available\.  
 This environment variable overrides `WARM_ENI_TARGET` behavior\.
 
+`MAX_ENI`
+Type: Integer
+Default: `-1`
```

Comment:
With both 0 and -1 being treated as using the max for the instance I would prefer we use 0 as the default but am open to discussion.